docs/changelog.md

```diff
@@ -161,10 +161,6 @@ after a shiny new JanusGraph header.
     
     `org.apache.tinkerpop.gremlin.server.GremlinServer` => `org.janusgraph.graphdb.server.JanusGraphServer`
 
-##### Drop support for Ganglia metrics
```

Comment:
You shouldn't remove it from the changelog that we are dropping support for ganglia.

Reply:
So you would like the changelog untouched?

Reply:
I will create a new pull request with those changes for tomorrow.

Reply:
You could update this PR.

Reply:
@jiyoungsin — once you update your branch locally, just run `git push -f` on your branch to update this PR in-place, no need to create a new PR to address comments.

Reply:
@mbrukman thank you for the information! I have pushed a new release. Please let me know if there are any other changes you would like me to complete before merging. I would also like to thank this community for the opportunity to contribute.

Reply:
I think i miss understood what i was supposed to do. Was I supposed to just re commit with the. "[doc only]" message?